src/components/LaatijaUpload/Patevyystaso.svelte

```diff
@@ -0,0 +1,46 @@
+<script>
+  import * as R from 'ramda';
+  import { locale, _ } from '@Language/i18n';
+  import * as Either from '@Utility/either-utils';
+  import * as Future from '@Utility/future-utils';
+  import * as Fetch from '@Utility/fetch-utils';
+  import { patevyystasoStore } from '@/stores';
+
+  export let value;
+
+  const patevyystasoFuture = R.compose(
+    Future.coalesce(Either.Left, Either.Right),
+    Fetch.responseAsJson,
+    Future.encaseP(Fetch.getFetch(fetch))
+  )('api/private/patevyydet/');
+
+  // TODO: Get patevyydet only once when store is empty
+  Either.isRight($patevyystasoStore) ||
+    Future.fork(
+      patevyystasoStore.set,
+      patevyystasoStore.set,
+      patevyystasoFuture
+    );
+
+  $: Either.isRight($patevyystasoStore) ||
+    R.compose(
+      Future.fork(patevyystasoStore.set, patevyystasoStore.set),
+      R.chain(Future.after(1000))
+    )(patevyystasoFuture);
```

Comment:
The way to handle these might change in near future. I'm working on it while working on Laatija-components.